source/instructors/setupcourses/virtualmachine.rst

```diff
@@ -2,37 +2,37 @@
    :description: Instructions for using Computed VMs.
 
 
-.. _computedvm:
+.. _virtualmachine:
 
-Computed VMs
-============
+Virtual Machines
+================
 
-Codio supports Windows OS in **Computed VM** boxes. To enable this for your organization, send an email to help@codio.com with approximate details of the number of students who will require access to computed VMs.
+Codio supports Windows and Ubuntu OS in **Virtual Machine** boxes. To enable this for your organization, send an email to help@codio.com with approximate details of the number of students who will require access to computed VMs.
 
 .. Note:: Codio is not responsible for licensing issues in regard to any software you install to use.
 
-The computed VM's can support 1, 4, 8 or 16Gb memory. VM's are enabled for the organization and then set up at the **Course** level. All assignments in the course can utilize the computed VM boxes, so we would recommend setting up a new course for those assignments that require them.  All assignments in the course would use the same **VM Stack**.
+The virtual machines can support 1, 4, 8 or 16Gb memory. VM's are enabled for the organization and then set up at the **Course** level. All assignments in the course can utilize the virtual machines boxes, so we would recommend setting up a new course for those assignments that require them.  All assignments in the course would use the same **VM Stack**.
 
 
-Computed VM's are not available for individual projects (ie when created in **My Projects** area).
+Virtual machines are not available for individual projects (ie when created in **My Projects** area).
 
  **Please note:** we have plans to develop this feature in the future but if you have ideas/suggestions please raise in our `Feedback area <https://feedback.codio.com/>`_
 
 
-When Computed VM's have been enabled for your organization, follow these steps:
+When virtual machines have been enabled for your organization, follow these steps:
 
 Enabling VM for the Course
 **************************
 
 - On the **Courses** page, select the course and then click the **Course Details** tab.
 
-- In the **Computed VM** section, toggle the **Enable Computed VM** button to **On**.
+- In the **Virtual Machine** section, toggle the **Enable Virtual Machine** button to **On**.
 
-- Select **OS type** (currently supported: Windows OS).
+- Select **OS type** (currently supported: Windows & Ubuntu).
 
 - Select **VM type** to allocate the memory required.
 
-- Select **VM Stack** - the default is **Windows Codio Ami** but once you have published your own VM Stack (see below), you will be able to select those as required.
+- Select **VM Stack** - select either **Windows Codio Ami** or **Ubuntu Codio Ami** depending on the OS type selected above but once you have published your own VM Stack (see below), you will be able to select those as required.
```

Comment:
Same comment - potential name change for the prompt.

Reply:
as above